Makefile-docker

```diff
@@ -115,8 +121,8 @@ install_python_test_dependencies:
 	$(PIP_COMMAND) install --progress-bar=off --no-deps --exists-action=w -r requirements/prod_py${PYTHON_VERSION_MAJOR}.txt
 	$(PIP_COMMAND) install --progress-bar=off --no-deps --exists-action=w -r requirements/tests.txt
 
-install_python_dev_dependencies: install_python_test_dependencies
-	$(PIP_COMMAND) install --progress-bar=off --no-deps --exists-action=w -r requirements/flake8.txt
+install_python_dev_dependencies:
```

Comment:
I really thought we needed the `install_python_test_dependencies` recipe command too in `install_python_dev_dependencies`

Reply:
Indeed, I was playing around with makefile locally and removed the dependency for faster test-cycle, I added it back in c35ee83